### ISSUE TITLE
2.3-beta4: broken bbb-libreoffice-docker

### ISSUE THREAD

**Issue by joo4mart** [bigbluebutton/bigbluebutton#12064]
**Describe the bug**
While running upgrade (with apt) from BBB 2.3 beta3 to beta4 the upgrade of bbb-libreoffice-docker failed.
The complete BBB is not useable anymore.

**To Reproduce**
1. starting: apt-get update
2. starting: apt-get dist-upgrade
   this gives error messages (see Screenhots)
3. try to repair with script: install-bbb.sh
   this gives again error messages (see Screenhots)

**Expected behavior**
All core packages should be upgraded without errors.

**Actual behavior**
The upgrade is broken for named package, BBB is not usable anymore.

**Screenshots**
```
Setting up bbb-libreoffice-docker (1:2.3.0-43) ...
#
# Building bbb-libreoffice docker image
#
Sending build context to Docker daemon  2.048kB
Step 1/6 : FROM openjdk:11-jre
 ---> 1cf688a75b44
Step 2/6 : ENV DEBIAN_FRONTEND noninteractive
 ---> Using cache
 ---> a72279c435be
Step 3/6 : RUN apt update
 ---> Using cache
 ---> 7f8658a73aa7
Step 4/6 : RUN apt -y install locales-all fontconfig libxt6 libxrender1
 ---> Running in 72ddceefe993

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Reading package lists...
Building dependency tree...
Reading state information...
Package locales-all is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'locales-all' has no installation candidate
E: Unable to locate package libxt6
E: Unable to locate package libxrender1
The command '/bin/sh -c apt -y install locales-all fontconfig libxt6 libxrender1' returned a non-zero code: 100
dpkg: error processing package bbb-libreoffice-docker (--configure):
 installed bbb-libreoffice-docker package post-installation script subprocess returned error exit status 100
```

**Desktop (please complete the following information):**
 - OS: Linux, Ubuntu 18.04.5 LTS
 - BBB: 2.3-beta3 to 2.3-beta4

**Additional context**
Perhaps the way using apt for upgrade is false? If it is so, what is the best way for upgrade?

**Comment by ffdixon** [comment]
Were you able to run the initial `bbb-install.sh` command again (it should be in your bash history).  After running that command do you see the same error?

**Comment by joo4mart** [comment]
Yes, I tried removing bbb-libreoffice-docker package, but don't help.
Then I tried running bbb-install.sh with all my local options, but don't help again.

**Comment by joo4mart** [comment]
In the release notes I have found:

Note - if upgrading from previous 2.3-beta install, edit /var/lib/dpkg/info/bbb-libreoffice-docker.prerm to comment out the line (near the bottom)
```
docker network remove bbb-libreoffice -> # docker network remove bbb-libreoffice
```
 this should help you to uninstall the previous bbb-libreoffice-docker version.

**Comment by joo4mart** [comment]
I have found a solution:

1. list all docker images and find an empty line:

```
$ sudo docker images

REPOSITORY                 TAG           IMAGE ID       CREATED         SIZE
                                         381ce13db8fa   4 days ago      690MB
postgres                   13.2-alpine   e43172f9204f   3 days ago      160MB
bigbluebutton/greenlight   v2            b57276099177   3 days ago      240MB
openjdk                    11-jre        1cf688a75b44   7 days ago      301MB
```

2. delete the empty docker image using IMAGE_ID, in my example:
`
$ sudo docker image rm -f 381ce13db8fa
`
3. running again script `install-bbb.sh` with all your local options. Finally running without errors.

**Comment by ffdixon** [comment]
We just released BigBlueButton 2.3-beta-5.  Can you upgrade and try it now and let us know if the conversion works.

**Comment by joo4mart** [comment]
I just started again `apt-get dist-upgrade` which upgrade these packages:
```
bbb-config
bbb-libreoffice-docker
```
Upgrade runs without any errors - perferctly! **And BBB 2.3 works again!**
